nx-dev/nx-dev/project.json

```diff
@@ -18,6 +18,12 @@
         ],
         "parallel": false
       },
+      "inputs": [
+        "production",
+        "^production",
+        "{workspaceRoot}/scripts/tsconfig.scripts.json",
+        "{workspaceRoot}/scripts/documentation/internal-link-checker.ts"
+      ],
```

Comment:
Updated inputs because the link checker was being cached even though I had change the script code